src/hooks/useCoin.ts

```diff
@@ -0,0 +1,18 @@
+import { CoinI } from "@/interfaces/coin.interface";
+import { useEffect, useState } from "react"
+
+export const useCoin = (symbol: string) => {
```

Comment:
hook naming looks like kakashka
it should be more understandable what this hook does

if it loads coin data then name it smth like `useGetCoinData` or `useLoadCoinData`
but it